google-cloud-no-speech/src/intents/handlers/NoInputIntentHandler.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const { NO_INPUT_EVENT_NAMES } = require('../GenesysNoInputEventNames');
+const { NO_INPUT_COUNT, NO_INPUT_LIMIT } = require('../GenesysParameterNames');
+
+class NoInputIntentHandler {

```

Comment:
I wonder if we could create a base class for an intent handler, and then inherit NoInputIntentHandler and DefaultIntentHandler from the base class.

Reply:
We probably could, but being that this is just a simple example, I don't know that we really need to go that in depth.